### DISCUSSION TITLE
Disable WiFi scan for wired cameras

### DISCUSSION THREAD

**Discussion by GqSteve** [roleoroleo/yi-hack-Allwinner-v2#436] [General]
I have an ESCAM PT202 that is connected via wire. The camera still scans for an AP...from initial setup procedure. The camera actually sends packages looking for the known AP. I had to change the password on the AP, otherwise the camera would also establish a connection over WiFi...that I don't want.

Is there a way to disable the WiFi connection, since a wired connection is working?

**Comment by roleoroleo** [comment]
You could try to comment the line
`ifconfig wlan0 up`
in /tmp/sd/lower_half_init.sh
Let me know if it works.

**Comment by GqSteve** [comment]
I am using the b091qp firmware and the line is already comment in the file:

```ifconfig ${NETWORK_IFACE} up
ethmac=d2:`ifconfig ${NETWORK_IFACE} |grep HWaddr|cut -d' ' -f10|cut -d: -f2-`
#if [ "${enable_4g}" = "y" ];then
#    ifconfig usb0 up
#    ethmac=d2:`ifconfig usb0 |grep HWaddr|cut -d' ' -f10|cut -d: -f2-`
#else
#    ifconfig wlan0 up
#    ethmac=d2:`ifconfig wlan0 |grep HWaddr|cut -d' ' -f10|cut -d: -f2-`
#fi
```

**Comment by GqSteve** [comment]
I shut down wlan0 with "ifconfig wlan0 down", but some time later the interface came up again. 

I see a  process "/backup/tools/wpa_supplicant -c/tmp/wpa_supplicant.conf -g/var/run/wpa_supplicant-global -Dnl80211 -iwlan0 -B". 
Could it be that this brings up the wlan0 again? How can I disable this process?

**Reply by roleoroleo** [reply]
The wireless lan is managed by yi processes and I don't know how to disable them.
I already tried, in the recent past, to use the wlan as an AP but I had to give up because I am not able to control the interface.
I think it's not possible to disable the wifi interface.

**Reply by GqSteve** [reply]
I killed the process for "/backup/tools/wpa_supplicant -c/tmp/wpa_supplicant.conf -g/var/run/wpa_supplicant-global -Dnl80211 -iwlan0 -B" and set the interface wlan0 down....yesterday....but today the process and the interface came up :-(

**Reply by roleoroleo** [reply]
Probably the process is "dispatch".

**Reply by GqSteve** [reply]
I cleared the SSID and password from the memory:

cat /dev/zero | dd of=/dev/mtdblock7 bs=1 seek=28 count=64 conv=notrunc
cat /dev/zero | dd of=/dev/mtdblock7 bs=1 seek=92 count=64 conv=notrunc

and rebootet the cam. Afterwards the cam didn't broadcast anything through the waln0 searching for the known SSID and the wpa_supplicant process hasn't been triggered until now. It seems indeed that the wpa_supplicant process is "dispatched" as long as a SSID configuration exists.
